Exercise_4.py

```diff
@@ -8,17 +8,25 @@ def __init__(self, data):
           
 """ Inorder traversal of a binary tree"""
 def inorder(temp): 
-  
```

Comment:
This implementation has a small error. Please try to fix it else you can reach out to me for any questions.